docs/articles_en/learn-openvino/openvino-samples/benchmark-tool.rst

```diff
@@ -226,8 +223,12 @@ should be used purposefully. For more information, see the
 
 .. note::
 
-   If the latency or throughput hint is set, it will automatically configure streams
-   and batch sizes for optimal performance based on the specified device.)
+   * If the latency or throughput hint is set, it will automatically configure streams
```

Comment:
```suggestion
   * If either the latency or throughput hint is set, it will automatically configure streams
```
is it "stream batch sizes" or "stream's batch sizes" "streams AND batch sizes" like in the original? Let's have some confirmation.

Reply:
It's streams and batch sizes, so a comma is in order:

```suggestion
   * If either the latency or throughput hint is set, it will automatically configure streams,
```